flight_computer/modules/altimeter/altimeter.py

```diff
@@ -0,0 +1,70 @@
+from machine import I2C, Pin, SPI
+from time import sleep
+import uos
+import bmp280
+import sdcard
+
+# BOARD DEFNS
+I2C_SDA_PIN = 6
+I2C_SCL_PIN = 7
+
+SPI_SCK_PIN = 2
+SPI_MOSI_PIN = 3
+SPI_MISO_PIN = 4
+SPI_CS_PIN = 5
+
+# ALTIMETER (BMP280) INIT (I2C)
```

Comment:
these should be in the classes respectively, i would suggest having a class per module because radio task needs the SD functionality but rn its embedded with the altimeter